modules/ROOT/partials/plugins/tinymcespellchecker-content-langs.adoc

```diff
@@ -11,8 +11,8 @@ include::partial$configuration/content_langs_base.adoc[]
 The Spell Checker Pro plugin will use the `code` property to determine which dictionary to use on the content, unless the `customCode` property is available. The language tag specified by `customCode` overrides the spelling dictionary, to allow for non-standard languages such as the medical dictionaries provided with Spell Checker Pro. For all languages, ensure that:
```

Comment:
This file needs an id added to the option title. e.g:
```
[[content_langs]]
=== `content_langs`
```

Reply:
Done